iam-login-service/src/main/java/it/infn/mw/iam/api/client/registration/service/DefaultClientRegistrationService.java

```diff
@@ -397,32 +399,36 @@ public RegisteredClientDTO updateClient(String clientId, RegisteredClientDTO req
 
```

Comment:
I'd move the check if oldClient is active here. If not simply throw the ClientSuspended exception.
Otherwise simply go on after the if.

Reply:
Done